pipelines/broad/reprocessing/cram_to_unmapped_bams/CramToUnmappedBams.wdl

```diff
@@ -153,7 +158,7 @@ task CramToBam {
   runtime {
     docker: "us.gcr.io/broad-gotc-prod/genomes-in-the-cloud:2.4.7-1603303710"
```

Comment:
This could use a specific samtools docker.  I *think* there's one in Skylab?  Not sure it's worth making that change now - just a thought.

Reply:
Skylab has one, as does the GDC pipeline, but they are not in gcr. Going to leave this for the docker overhaul